crates/rsonpath-lib/src/query/error.rs

```diff
@@ -39,7 +39,7 @@ use std::{
 };
 use thiserror::Error;
 
-use super::ARRAY_INDEX_ULIMIT;
+use super::nonnegative_array_index::ARRAY_INDEX_ULIMIT;
```

Comment:
Please avoid adding newlines between `use` statements, it breaks standard `rustfmt` sorting of uses.

Reply:
Ill try to be mindful of this, thanks for the heads up.